pkg/io/video/convert_cgo.go

```diff
@@ -14,27 +15,35 @@ import "C"
 // All functions switched at runtime must be declared also in convert_nocgo.go.
 const hasCGOConvert = true
 
-func i444ToI420(img *image.YCbCr) {
+func i444ToI420(img image.YCbCr) image.YCbCr {
 	h := img.Rect.Dy()
+	cLen := img.CStride * h / 4
+	cbDst, crDst := make([]uint8, cLen), make([]uint8, cLen)
```

Comment:
I tried to use a 0 len here with a cLen capacity to avoid the overhead of filling slice with empty values.

But I didn't found any way to get the data stored by the C code. Keep having index out of range, and didn't found any way to set the slice len without losing the data.

Reply:
`sync.Pool` should reduce such overhead.

Reply:
I will try to see if it helps to speed up things

Reply:
Ok, I have tried to use Pool, but since the goal is to avoid the share of the slice between runs, I didn't find any way to use it to improve performance. It always involve at some point making a slice of the right length.

Reply:
Perhpas we can merge this as is and we will see in the future if we find any good solution ?
When the release function will be actually used for example ?

Reply:
Release function in example:
https://github.com/pion/mediadevices/blob/43272ea9658e2b75b7e382e9f71d20f1e39ece2b/examples/http/main.go#L62-L71

I think it's ok to merge and optimize later.